include/itkCuberilleImageToMeshFilter.hxx

```diff
@@ -745,18 +752,22 @@ CuberilleImageToMeshFilter<TInputImage, TOutputMesh, TInterpolator>::CalculateLa
   //
   //  }
   //
-  //  for (size_t i = 0; i < pow(2,8); ++i) {
+  //  // Manually handle the corner cases, discussed above.
+  //
+  //  this->m_LabelsArray[126] = { -1, 0, 0, 1, 0, 1, 1, -1 };
```

Comment:
Should this manual initialization block come after the for loop?

Reply:
@dzenanz The manual initialization should come _between_ the two for loops, as shown.  The first for loop calculates the array, the initialization block then "corrects" the four cases which are incorrect by this method, and the second for loop outputs the hardcoded c++ array.  I hope this is clear, but please let me know if not!

Reply:
This PR didn't touch the generating loop, so I didn't notice it. Sorry for noise.